### DISCUSSION TITLE
Are there any plans to upgrade to support ng13 in the near future?

### DISCUSSION THREAD

**Discussion by ynsunxp** [docgeni/docgeni#318] [General]
This is a very awesome project! ~~But it failed when I added it to the ng13 project.~~

**Comment by ynsunxp** [comment]
OOP!  It's my usage that's wrong. Please Remove the discussion. @why520crazy

**Reply by why520crazy** [reply]
What's wrong? sorry, now I see this.

There is a bug when exec `docgeni init` and `ng add @docgen/cli`. you can add docgeni to your project manual.

The template example has been upgraded angular to 13.x, see https://github.com/docgeni/docgeni-template
